tox.ini

```diff
@@ -0,0 +1,87 @@
+[tox]
+envlist = bandit,black,isort,flake8,python3.9,python3.10
+
+[gh-actions]
+python =
+    3.9: python3.9
+    3.10: python3.10
+
+[testenv]
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+usedevelop = true
+
+commands =
+    pytest \
+        --cov-config .coveragerc --cov=pyarn --cov-report term
+
+[testenv:black]
+description = black checks [Mandatory]
+skip_install = true
+deps =
+    # Pin the version of black to avoid a newer version causing tox to fail
+    black==22.6.0
+commands =
+    black --check --diff pyarn tests
+
+[testenv:isort]
+skip_install = true
+profile = "black"
+deps =
+    isort[colors]
+commands =
+    isort --check --diff --color pyarn tests
+
+[testenv:flake8]
+skip_install = true
+deps =
+    flake8==3.9.2
+    flake8-docstrings==1.6.0
+commands =
+    flake8
+
+[flake8]
+show-source = True
+max-line-length = 100
+exclude = venv,.git,.tox,.env,pyarn/parsetab.py,pyarn/parser.out
+ignore = D101,D102,D103,D104,D105,D107,D300,D400,E226
+; Missing Docstrings
+; D101	Missing docstring in public class
+; D102	Missing docstring in public method
+; D103	Missing docstring in public function
+; D104	Missing docstring in public package
+; D105	Missing docstring in magic method
+; D107	Missing docstring in __init__
+
+; Quotes Issues
+; D300	Use “”“triple double quotes”“”
+
+; Docstring Content Issues
+; D400	First line should end with a period
+
+; Whitespace Issues
+; E226	Missing whitespace around arithmetic operator
+
+per-file-ignores =
```

Comment:
Here I would also change the comments to be in the lines before the actual declaration, and follow the indent level.
```
per-file-ignores =
    ; Whitespace Issues
    ; E231	Missing whitespace after ',', ';', or ':'
    ; E241	Multiple spaces after ','
    tests/*:E231,E241
    
    ; D205	1 blank line required between summary line and description
    ; D208	Docstring is over-indented
    ; D209	Multi-line docstring closing quotes should be on a separate line
    ; D403	First word of the first line should be properly capitalized
    ./pyarn/parser.py:D205,D208,D209,D403
```